### PR TITLE
Portfolio mobile version second milestone pull request

### PR DESCRIPTION
**Pull request by Ahsan12356**
It's the second milestone of the project "[Portfolio: setup and mobile version skeleton](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)" in which we had to create the middle section of the Figma portfolio template.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -76,5 +80,276 @@ body {
...
+  margin-bottom: 292px;
+}
+
+.middle {
+  display: grid;
+  grid-template-rows: auto auto;
+  width: 373px;
+  background-color: #fff;
+  border: 1px solid #fff;
+}
```
Comment (mirouhml) [inline style.css]: - [x] Please change the `background-color` here to match the design of the Figma template (it should be `#e5e5e5`) 😁

File: style.css [hunk 2]
```diff
@@ -76,5 +80,276 @@ body {
...
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background-color: transparent;
+  cursor: pointer;
+  margin-bottom: 28px;
+}
```
Comment (mirouhml) [inline style.css:210]: - [ ] Please add styling for the other states of the buttons (`active`, `hover` and `disabled`) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design: 😁

image

File: index.html [hunk 3]
```diff
@@ -27,5 +27,110 @@ <h1>I'm Ahsan Ghazni Khan!</h1>
             <li><a href=""><img src="images/rabbit.png" alt=""></a></li>
         </ul>
     </section>
+    <section class="middle">
+        <div class="cards">
+            <div class="card">
+                <img src="images/first_card_image.png" alt="">
+                <h1>Tonic</h1>
```
Comment (mirouhml) [inline index.html]: - [x] Please know that the `h1` tag should be used only once on your webpage as it serves as a main title, and you already used it for your intro (which is the right place to use it), kindly change this to another `hx` tag. 😁

File: index.html [hunk 4]
```diff
@@ -27,5 +27,110 @@ <h1>I'm Ahsan Ghazni Khan!</h1>
...
+                    <li><p>css</p></li>
+                    <li><p>javaScript</p></li>
+                </ul>
+                <button class="btn see-projects">See Projects</button>
+            </div>
+        </div>
+    </section>
+    <section class="about_myself_section">
+        <div class="about_myself">
+            <h1>About Myself</h1>
```
Comment (mirouhml) [inline index.html]: - [x] Same comment here about the `h1` tag, kindly change this to another `hx` tag. 😁

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Ahsan12356, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Ahsan12356) [reply style.css]: Done

File: index.html (hunk 3 above)
Comment (Ahsan12356) [reply index.html]: Resolved

File: index.html (hunk 4 above)
Comment (Ahsan12356) [reply index.html]: Resolved. Thanks for notifying.

**Review by zhadier — APPROVED** [review]
## Hi @Ahsan12356 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**